docs: run full node

restorenode opened the pull request:
Updating files in `full-node/run-a-full-terra-node`.

restorenode committed:
docs: run full node

Updating files in `full-node/run-a-full-terra-node`.

Jared-TFL commented on docs/full-node/run-a-full-terra-node/set-up-production.mdx:
```diff
@@ -10,13 +10,13 @@ For information about running a validator node, visit the [validator guide](../m
 ## Increase the maximum files Terrad can open
 
-Terrad is set to open 1024 files by default. It is recommended that you increase this amount.
+Terrad can open 1024 files at a time by default. It is recommended that you increase this amount.
```
I think this is linux, not Terrad.
Terrad can open a ton of files. I set one node to over 1,000,000 open files allowed.

restorenode replied:
@Jared-TFL Nice catch!  I was focusing on the grammar and didn't catch this one.

Jared-TFL commented on docs/full-node/run-a-full-terra-node/updates-and-additional.mdx:
````diff
@@ -20,7 +20,7 @@ rm ~/.terra/config/addrbook.json
 ```
 
-Your node is now in a pristine state, keeping the original `priv_validator.json` and `config.toml`. If you had any sentry nodes or full nodes set up before, your node will still try to connect to them but may fail if they haven't also been upgraded.
+Your node is now in a pristine state, keeping the original `priv_validator.json` and `config.toml`. If you had any sentry nodes or full nodes set up before, your node will still try to connect to them, but may fail if they haven't also been upgraded.
````
This sounds a little misleading.
Terrad tendermint unsafe reset all will delete the private key, and this will lead to a tombstoned validator.

restorenode committed:
fix: Upgrade the testnet steps

Alert the user that they will have to make backups of the `priv_validator.json` and `cargo.toml` files before reseting their data.

evanorti commented on docs/full-node/run-a-full-terra-node/configure-general-settings.mdx:
```diff
@@ -70,9 +70,8 @@ For information about the available Terra REST API endpoints, see the [Swagger d
-   # Enable defines if the API server should be enabled.
-   enable = true
+    # Enable defines if the API server should be enabled.
+    enable = true
```
I think this section is broken due to misplaced back tics. Can you try it out locally and see what the issue is? I think line 62 is missing the close of the code block.

evanorti replied:
resolved.

evanorti commented on docs/full-node/run-a-full-terra-node/join-a-network.mdx:
```diff
@@ -38,7 +38,7 @@ The simplest Terra network you can set up is a local testnet with just a single
 <Admonition type="tip" icon="💡" title="Get tokens">
 
-In order for Terrad to recognize a wallet address, it must contain tokens. For the testnet, use [the faucet](https://faucet.terra.money/) to send Luna to your wallet. If you are on mainnet, send funds from an existing wallet. 1-3 Luna are sufficient for most setup processes.
+In order for Terrad to recognize a wallet address, it must contain tokens. For the testnet, use [the faucet](https://faucet.terra.money/) to send Luna to your wallet. If you are on mainnet, send funds from an existing wallet. 1-3 Luna is sufficient for most setup processes.
```
Since 1-3 is indicative of a plural amount, it should be "1-3 Luna are sufficient..."

For example, you'd say "1-3 cars are sufficient" vs "1-3 cars is sufficient."

evanorti replied:
resolved.

evanorti committed:
Update configure-general-settings.mdx

evanorti committed:
Update join-a-network.mdx
